Gemfile

```diff
@@ -51,3 +51,5 @@ end
 
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
+# add gem haml
+gem 'haml-rails'
```

Comment:
When you added a gem - please pin down its version to prevent bugs w/ version compatibility.